src/client/utils/styles/_variables.scss

```diff
@@ -30,63 +26,83 @@ $danger: #e24c4c;
 $positive: #08b157;
 $negative: #e14040;
 
-// Colors -------------------
-
-$brand:                   hsl(21, 100%, 50%);
-$dark:                    #004584;
-
-$error:                   hsla(0, 72%, 59%, 1.00);
-
-$border-super-light:      #e6e6e6;
-$border-extra-light:      hsl(0, 0%, 87%);
-$border-light:            hsl(0, 0%, 82%);
-$border-medium:           hsl(0, 0%, 73%);
-$border-dark:             hsl(0, 0%, 60%);
-$border-darker:           hsl(0, 0%, 40%);
-
-$background-base:         #e1e1e1;
-$background-light:        hsl(0, 0%, 93%);
-$background-lighter:      hsl(0, 0%, 95%);
-$background-lightest:     hsl(0, 0%, 96%);
-$background-medium:       hsl(0, 0%, 87%);
-$background-dark:         hsl(0, 0%, 55%);
-
-$background-brand:        $brand;
-$background-brand-light:  hsl(21, 100%, 70%);
+// Theme Colors
+
+$brand: hsl(21, 100%, 50%);
+$background-base: #e1e1e1;
+$error: hsla(0, 72%, 59%, 1.00);
+$hover: hsl(21, 100%, 70%);
+$text-standard: hsl(0, 0%, 13%);
+$item-dimension: hsl(21, 100%, 80%);
+$item-measure: hsl(205, 60%, 77%);
+$main-time-line: rgb(202,	236,	230);
+
+$variables: (
+  background-base: $background-base,
```

Comment:
I think `-color` suffix will show that all variables are directly related to color. Now for example background-base does not show wich property it includes.

Reply:
Renaming `$variables` to something like `$themeColors` would have the same benefit but is more succinct. As long as there are only colors inside the map.

Reply:
I don't have a strong opinion on which solution would be better. We should consider what happens if we add additional variables that determine other aspects of the theme like distances, width or text weight.

Adding the color suffix would make variable names longer but would keep all variables in the same place.
Renaming the map to `$themeColors` would not be a big change right now, but if we want to add other kinds of variables we would need to differentiate which `map` to use. This could be done explictly by passing the used map into the mixin (or adding a mixin per map) or by auto detecting the appropriate map for the given CSS property. E.g. for `color` use `$themeColors` but for `font-weight` use `$themeTypography`.

Reply:
@erykpiast @bigsergey any preference here?

Reply:
$themeColors and $themeTypography look good for me.